bcda/cclf/local_fileprocessor.go

```diff
@@ -123,13 +128,52 @@ func (p *processor) walk(path string, info os.FileInfo, err error) error {
 			continue
 		}
 
-		key := metadataKey{perfYear: metadata.perfYear, fileType: metadata.fileType}
-		sub := p.cclfMap[metadata.acoID]
-		if sub == nil {
-			sub = make(map[metadataKey][]*cclfFileMetadata)
-			p.cclfMap[metadata.acoID] = sub
+		if metadata.cclfNum == 0 {
+			if cclf0Metadata != nil {
+				readError = fmt.Errorf("Multiple CCLF0 files found in zip (%s)", path)
+				break
+			}
+			cclf0Metadata = &metadata
+			cclf0File = f
+		} else if metadata.cclfNum == 8 {
+			if cclf8Metadata != nil {
+				readError = fmt.Errorf("Multiple CCLF8 files found in zip (%s)", path)
+				break
+			}
+			cclf8Metadata = &metadata
+			cclf8File = f
+		} else {
+			readError = fmt.Errorf("Unexpected CCLF num %d processed (%s)", metadata.cclfNum, path)
+			break
 		}
-		sub[key] = append(sub[key], &metadata)
+	}
+
+	if readError != nil {
+		p.failure++
+		println(readError.Error())
```

Comment:
will this println cause any issues with our log output? ie printing in non-json output?

Reply:
This is the local fileprocessor which is only used locally and in tests (and used to be used in Jenkins smoke tests for importing synthetic CCLF files.) Shouldn't end up in Splunk for the most part.